examples/scheduling.rst

```diff
@@ -105,7 +105,7 @@ The next code lines create a constraint from this function and adds it to CSP in
 >>> csp = dwavebinarycsp.ConstraintSatisfactionProblem(dwavebinarycsp.BINARY)
 >>> csp.add_constraint(scheduling, ['time', 'location', 'length', 'mandatory'])
 
-This tool can also convert the binary CSP to a BQM. The following code does so and
+:std:doc:`dwavebinarycsp <binarycsp:index>` can also convert the binary CSP to a BQM. The following code does so and
```

Comment:
Sure, but please add commas: This tool, :std:doc:`dwavebinarycsp <binarycsp:index>`, can also convert

Reply:
Fixed.